lib/services/jellyfin_api_helper.dart

```diff
@@ -59,6 +60,12 @@ class JellyfinApiHelper {
   static Future<void> _processRequestsBackground((SendPort, RootIsolateToken) input) async {
     BackgroundIsolateBinaryMessenger.ensureInitialized(input.$2);
     ReceivePort requestPort = ReceivePort();
+
+    // Extend the default security context to trust Android user certificates.
+    // This is a workaround for <https://github.com/dart-lang/sdk/issues/50435>.
+    final _flutterUserCertificatesAndroidPlugin = FlutterUserCertificatesAndroid();
+    await _flutterUserCertificatesAndroidPlugin.trustAndroidUserCertificates(SecurityContext.defaultContext);
```

Comment:
Unfortunately, Dart isolates don't seem to inherit the `SecurityContext.defaultContext` we configured at app startup. I don't know if there's a cleaner way of handling this.

Reply:
Yes, that's good enough :)